cluster/charts/crossplane/values.yaml.tmpl

```diff
@@ -59,3 +59,6 @@ securityContextRBACManager:
 alpha:
   oam:
     enabled: false
+
+metrics:
+  enabled: false
```

Comment:
@hasheddan I put the exposure of metrics port and endpoint behind this variable and I opted for being disabled by default. Although having it enabled by default is also nice to have too. I think I'm fine either way.